addons/web/views/webclient_templates.xml

```diff
@@ -409,11 +409,16 @@
                     <t t-esc="message"/>
                 </p>
 
-                <input type="hidden" name="redirect" t-att-value="redirect"/>
                 <div t-attf-class="clearfix oe_login_buttons text-center mb-1 {{'pt-2' if form_small else 'pt-3'}}">
                     <button type="submit" class="btn btn-primary btn-block">Log in</button>
+                    <t t-if="debug">
+                        <em t-attf-class="d-block text-center text-muted small ">- or -</em>
+                        <button type="submit" name="redirect" value="/web/become" class="btn btn-block btn-warning">Log in as superuser</button>
```

Comment:
As we have "become " in the debug menu i would use the alternative more sober look.

Reply:
I prefer the more sober look too 👍 ... but it will be in 2 lines if "password reset" is active.
<img width="387" alt="screen shot 2018-09-27 at 07 41 58" src="https://user-images.githubusercontent.com/8558351/46125358-064da680-c229-11e8-99e9-2c0eb62beba4.png">

It's ok but  I'm worried that it starts to be too complex down there... what about an even more sober design? Like add a little dropdown at the top-right?  It could group other advanced features too if we'll have to add them in the future...
<img width="421" alt="screen shot 2018-09-27 at 07 42 59" src="https://user-images.githubusercontent.com/8558351/46125417-44e36100-c229-11e8-9def-6571c8428957.png">